api/base/serializers.py

```diff
@@ -13,6 +12,27 @@
 from api.base.exceptions import InvalidQueryStringError, Conflict
 
 
+def format_relationship_links(related_link, self_link, rel_meta, self_meta):
+
+    ret = {
+        'links': {
+            'related': {
+                'href': related_link,
+                'meta': rel_meta
+            },
+            'self': {
+                'href': self_link,
+                'meta': self_meta
+            }
+        }
+    }
+
+    # Not currently returning self links
+    if not ret['links']['self']['href']:
```

Comment:
Since we're not really implementing `self`links, I didn't want a bunch of empty self objects cluttering the relationships.   But I thought it was useful to return an empty dict for the related link if one existed.  If there's no parent, then an empty dict in the response quickly tells me that.